### DISCUSSION TITLE
Actividades a entregar

### DISCUSSION THREAD

**Discussion by RicardoArquimedes** [Launch-X-Latam/MisionFrontEnd#444] [Team Posts]
Hola compañeros/as recién ingreso a github, me siento un poco atrasado y quisiera saber que actividades debemos tener listas a la fecha. Muchas gracias 🙂

**Comment by Armando9576** [comment]
Hola amigo, buen dia.
Pues hasta ahora tenemos la primera práctica, que honestamente no es gran cosa, pero si lleva su tiempo.
Te recomiendo que veas estas dos grabaciones:

https://web.microsoftstream.com/video/41b9721d-cde1-4ffd-bdf0-7a877d57e5e1

https://teams.microsoft.com/l/meetup-join/19%3ameeting_M2U1OTM0ZjQtYjQyMS00YTAwLThlNjItMTY1ZmU0MDNkYjg5%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%2296eeb66f-b984-4083-8ea6-74c34a07a34a%22%2c%22IsBroadcastMeeting%22%3atrue%7d


En esta última, en los 30 minutos finales, es donde se explican lo que se debe de hacer, pero te recomiendo veas ambas sesiones a profundidad, para mayor comprensión.

Y después de estas sesiones y practicas, enmpieza html.

https://teams.microsoft.com/l/meetup-join/19%3ameeting_YjM4MWVmYWEtOTU0Mi00YWFiLWI2MjItZDExZTVhNDk2NjU4%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%2296eeb66f-b984-4083-8ea6-74c34a07a34a%22%2c%22IsBroadcastMeeting%22%3atrue%7d

Y pues basicamnete es todo lo que llevabmos. mucha suerte y animos amigo.

**Comment by RicardoArquimedes** [comment]
Muchas gracias

**Comment by RicardoArquimedes** [comment]
@Armando9576 en donde debo presentar o publicar las evidencias de los requerimientos o tareas solicitadas?

**Comment by zaazilcb** [comment]
> @Armando9576 en donde debo presentar o publicar las evidencias de los requerimientos o tareas solicitadas?

Hola, hasta donde yo sé no han dicho en donde, pero nos pidió que trabajemos en la práctica por ahora y posteriormente diran en donde y como irlo publicando

**Comment by MrAlizarinCrimson** [comment]
Perfecto, gracias al compañero que facilitó la información, estaba en las mismas
